dune

```diff
@@ -42,3 +42,5 @@
  (name runtest)
  (package coqide-server)
  (deps test-suite/summary.log))
+
+; (dirs (:standard _build_ci))
```

Comment:
Is this again an unrelated change? You may want to remove it while rebasing.

Reply:
This is OK IMHO as it is in the overaly commit, and uncommenting it is what you need to test them with Dune.